src/components/rooms/rawusagedata.ts

```diff
@@ -0,0 +1,10 @@
+import { Group } from '../groups/group'
+
+/**
+ * The number of events in a room on a given day
+ */
+export class RawUsageData extends Group {
+  room: number
+  day: Date
+  many: number | string
```

Comment:
Maybe `count` instead of `many` would be a more descriptive name for this field. Also, I think this should be of type `number` instead of `number | string`

Reply:
Resolved in 7f14d2fa53574ba4f674a088b4cb920fe7baac6f
also, fixed the broken db query in 9238305d5f601a2edd8cda783f12561b12e0603d (but with reintroducing `raw` in the query)